ui/desktop/app/components/settings-card/server/index.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+
+export default class SettingsCardServerComponent extends Component {
+  // =services
+  @service intl;
+
+  // =attributes
+  /**
+   * Returns the provider name for HCP and generic name for non HCP servers
+   * @type {string}
+   */
+  get providerName() {
+    const regex = /hashicorp\.cloud/;
```

Comment:
Should this also take into account int and dev environments as well?

Reply:
good call, included regex to check for `boundary and hcp/hashicorp` in the URL string
```
.boundary.hcp.dev/
.boundary.hcp.to/
.boundary.hashicorp.cloud/
```